tap_s3_csv/transform.py

```diff
@@ -319,9 +319,9 @@ def _transform(self, data, typ, schema, path):
         elif typ == "integer":
             if isinstance(data, str):
                 data = data.replace(",", "")
-
             try:
-                return True, int(data)
+                int(data)
+                return True, str(data)
```

Comment:
feel like we should keep the original form of the data by kept it in string, due to confirmation screen, int(a) can discard leading zero etc

Reply:
hmmm I’m surprised this works. I would have thought the `int(data)` that you call strip the leading zeros before `str(data)` is able to give output that we want? Why not remove `int(data)` entirely? What purpose is it serving?

Reply:
int(data) make sure we can convert data into an int, otherwise should return False